src/sentry/api/endpoints/organization_member_index.py

```diff
@@ -166,22 +173,20 @@ def post(self, request, organization):
         if settings.SENTRY_ENABLE_INVITES:
             om.token = om.generate_token()
 
-        try:
-            with transaction.atomic():
-                om.save()
-        except IntegrityError:
```

Comment:
I removed the integrity check here because the validator now also checks if the email already exists. I'm not super worried about races here, would rather not duplicate the response message.